tests/e2e/steps.go

```diff
@@ -22,13 +22,9 @@ var happyPathSteps = concatSteps(
 	stepsRedelegateForOptOut("consu"),
 	stepsDowntimeWithOptOut("consu"),
 	stepsRedelegate("consu"),
-	stepsDowntime("consu"),
```

Comment:
No need to test for downtime twice, testing for downtime with soft opt-out also covers "normal" downtime.

Reply:
It will, but there's an additional issue that proposes changes to how this runs in the CI:
* https://github.com/cosmos/interchain-security/issues/1175

We can have multiple jobs running in parallel and make the execution time shorter